modules/oxide/src/less/theme/content/comments/comments.less

```diff
@@ -2,13 +2,47 @@
 // Comments
 //
 
-@document-comment-background-color: #fff0b7;
-@document-comment-active-background-color: #ffe168;
+@document-comment-color: #ffe89d;
+@document-comment-active-color: #fed635;
+@document-comment-background-color: @document-comment-color;
+@document-comment-active-background-color: @document-comment-active-color;
+@document-comment-outline-color: @document-comment-color;
+@document-comment-active-outline-color: @document-comment-active-color;
 
-.tox-comments-visible .tox-comment {
-  background-color: @document-comment-background-color;
-}
+.tox-comments-visible {
+  .tox-comment[contenteditable="false"],
+  span.tox-comment img,
+  span.tox-comment > audio,
+  span.tox-comment > video,
+  span.tox-comment span.mce-preview-object {
+    &:not([data-mce-selected]) {
+      outline: 3px solid @document-comment-outline-color;
```

Comment:
Should this width and the ones below be using the `@object-block-selected-outline-width` LESS variable so it keeps it as the same size as the selection? Either that or maybe we should have this at the top as a new variable? e.g
```
@document-comment-outline-width: @object-block-selected-outline-width;
```

Reply:
That's a good idea. Will make the change

Reply:
Done